internal/gatewayapi/translator_test.go

```diff
@@ -315,6 +317,76 @@ func TestTranslate(t *testing.T) {
 				},
 			)
 
+			svc := corev1.Service{
+				ObjectMeta: metav1.ObjectMeta{
+					// Matches proxy.ExpectedResourceHashedName()
+					Name:      fmt.Sprintf("%s-%s", config.EnvoyPrefix, utils.GetHashedName(string(translator.GatewayClassName), 48)),
```

Comment:
ref https://github.com/envoyproxy/gateway/pull/6597/files?file-filters%5B%5D=.go&show-viewed-files=true#r2241402007

you need to add service with correct name in gateway namespace mode.

Reply:
Fixed